docs: Add basic Kubernetes deployment instructions (#369)
kinvolk/nebraska: docs/authorization.md

Rest of the steps is the same.

# Deploying on Kubernetes using the Helm Chart

We maintain a Helm Chart for deploying a Nebraska instance to Kubernetes. The
Helm Chart offers flexible configuration options such as:

- Deploy a single-replica `PostgreSQL` database together with Nebraska. We use
  the container image and also the Helm Chart (as a subchart) from
  [Bitnami](https://github.com/bitnami/charts/tree/master/bitnami/postgresql)

> - Enable / disable and configuring persistence for both Nebraska and PostgreSQL

Reviewer: ```suggestion
- Enabling / disabling, and configuring persistence for both Nebraska and PostgreSQL
```

Reviewer: Also, what do you mean it disables persistency for Nebraska and Postgres separately?

Author: By default both the Nebraska Pods and PostgreSQL Pods are using ephemeral storage (`emptyDir`). All data written to this storage is lost once a Pod needs to be restarted on a different node. For Nebraska this is only relevant when using the `host-flatcar-packages` option. For PostgreSQL this would mean, you need to start over from a blank Nebraska instance.

  (persistence is disabled by default)

- Common deployment parameters (exposing through `Ingress`, replica count, etc.)

> - All [Nebraska application configration](https://github.com/kinvolk/nebraska/tree/master/charts/nebraska#nebraska-configuration)

Reviewer: ```suggestion
- All [Nebraska application configuration](https://github.com/kinvolk/nebraska/tree/master/charts/nebraska#nebraska-configuration)
```

For the complete list of all available customization options, please read the
[Helm Chart README](https://github.com/kinvolk/nebraska/blob/master/charts/nebraska/README.md).

To install the Helm Chart using the default configuration (noop authentication),

> you can simply execute:

Reviewer: ```suggestion
you can execute:
```
